kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -103,27 +105,68 @@ window.SidebarView = BaseView.extend({
 
         this.$('.sidebar-content').append(this.topic_node_view.el);
 
+        this.set_sidebar_back();
+
         return this;
     },
 
-    resize_sidebar: _.debounce(function() {
+    resize_sidebar: function() {
+        if ($(window).width() < 1260)
```

Comment:
I wonder if there is a better way to do this instead of pixels. 1260 was approximately the width of the fully extended sidebar with the tab on my screen at least. I'm assuming the sidebar is somehow scaled down on smaller devices? Curious to see how it works on a Nexus tablet.